static/css/topic_page.scss

```diff
@@ -60,10 +60,12 @@ $chart-container-top-margin: 20px;
   display: flex;
 }
 
-.left-tiles,
-.right-tiles {
+.block-column {
   display: flex;
   flex-direction: column;
+  padding: 0.5rem;
```

Comment:
Can we add this as a `gap` instead, to avoid an additional .5rem padding on the outer edges.

https://developer.mozilla.org/en-US/docs/Web/CSS/gap#flex_layout

Reply:
as discussed offline: row adds negative padding, so adding padding to the outer edges to get net 0 padding.